clang/lib/Headers/arm_acle.h

```diff
@@ -20,8 +20,8 @@
 extern "C" {
```

Comment:
Thanks, all of these updates look correct to me.
Since these all refer to sections in a document, I think it may be useful too to add a hyperlink at the top of the document to where the ACLE lives.
I think the correct hyperlink would be https://github.com/ARM-software/acle/releases/.

@vhscampos Do you think that would be a good idea too?

Reply:
Thanks @Blue-Dot for the patch. I agree with @kbeyls that it would be a good idea.

Reply:
You may want to note that section numbers were correct at time of writing, if they've changed then look for the section title.